lib/commands/register.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     register: function (renderer) {
         var done = arguments[arguments.length-1] || function () {};
 
-        renderer.log(chalk.yellow('This will publish this plugin to', Constants.Registry));
+        //renderer.log(chalk.yellow('This will publish this plugin to', Constants.Registry));
```

Comment:
Was this intentionally commented out?

Reply:
It was intentional. In the presence of `.bowerrc` the function `Constants.Registry` returns undefined. I know there is a better solution than just commenting out, but I don't have time at the moment unfortunately.